### PR TITLE
@alloy -> paginate when page controller is tapped

### PR DESCRIPTION
**Pull request by 1aurabrown**
fixes https://github.com/artsy/eigen/issues/369

UIPageControl is not built to handle skipping to a specific page, but rather just to move right or left depending which side you tap on. So that's all I added -- moving forward or backward by one page when tapped.

### CODE REVIEW

File: Artsy/Classes/View Controllers/ARHeroUnitViewController.m [hunk 1]
```diff
@@ -149,6 +157,18 @@ -(void)goToNextHeroUnit
...
+    UIViewController *nextVC = [self pageViewController:self.pageViewController viewControllerBeforeViewController:[self currentViewController]];
+    [self goToHeroUnit:nextVC withDirection:UIPageViewControllerNavigationDirectionReverse];
+}
+
+-(void)goToNextHeroUnit
+{
+    UIViewController *nextVC = [self pageViewController:self.pageViewController viewControllerAfterViewController:[self currentViewController]];
+    [self goToHeroUnit:nextVC withDirection:UIPageViewControllerNavigationDirectionForward];
+}
+
```
Comment (alloy) [inline Artsy/Classes/View Controllers/ARHeroUnitViewController.m]: What are these for?

**Comment by alloy** [comment]
Can you add a CHANGELOG?

File: Artsy/Classes/View Controllers/ARHeroUnitViewController.m (hunk 1 above)
Comment (1aurabrown) [inline Artsy/Classes/View Controllers/ARHeroUnitViewController.m]: `goToNext` is for the timer that automatically swipes through them. i thought i would use `goToNext` and `goToPrevious` for the paginator, but ended up not. I can take the `Previous` one out.

**Comment by alloy** [comment]
I rebased your branch and merged it 27372d25b9d57260abeb25622f7c124a3bed741a

**Comment by 1aurabrown** [comment]
Oh i rebased recently too, i guess it got stale again. Thanks!

On Tue, May 12, 2015 at 07:22, artsy/eigen
<reply@reply.github.com>
wrote:
Closed #420 [https://github.com/artsy/eigen/pull/420] .

**Pull request closed without merging**